app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -49,17 +44,36 @@ export default function NoteForm({ onClose, refetch, note, isEdit }) {
               rows={8}
               required
             />
+            <Select
+            required
+            isMulti
+            label="Assigned Contact"
+            name="assigned_contacts"
+            placeholder="Select Role"
+            className="w-full flex-grow-0"
+            options={ASSIGNED_CONTACT_OPTIONS}
+          />
```

Comment:
Had you bypassed the eslint git hooks? Because this formatting should've been caught by our eslint config within wheel. Lemme know why this is formatted like this and on whether you had bypassed the git hooks.